src/fn/fn-popover.scss

```diff
@@ -0,0 +1,136 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-popover;
+
+$fn-popover-arrow-size: 0.75rem;
+$fn-popover-padding: 0.75rem;
+$fn-popover-button-margin: 0.375rem;
+$fn-popover-arrow-offset: -0.25rem;
+$fn-popover-arrow-margin: 1.5rem;
+$fn-popover-arrow-middle-calc: calc(50% - 0.25rem);
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex(column);
+
+  background-color: $fn-color-white;
+  border-radius: $fn-border-radius-12;
+  padding: $fn-popover-padding;
+  position: relative;
+  filter: drop-shadow(0 0 0.125rem rgba(0, 0, 0, 0.25)) drop-shadow(0 0.5rem 1rem rgba(0, 0, 0, 0.25));
+
+  &--no-padding {
+    padding: 0;
+  }
+
+  &__title {
+    @include fn-reset();
+
+    padding: $fn-popover-padding;
+    width: 100%;
+
+    .#{$block}__text {
+      @include fn-reset();
+
+      font-size: 1.125rem;
+      font-weight: bold;
+      line-height: 1.375rem;
+
+      &--truncate {
+        @include fn-ellipsis();
+      }
+    }
+  }
+
+  &__content {
+    @include fn-reset();
+
+    margin: auto;
+    padding: $fn-popover-padding;
+  }
+
+  &__button-container {
+    @include fn-reset();
+    @include fn-flex();
+    @include fn-set-margin-left(auto);
+
+    padding: $fn-popover-padding $fn-popover-padding $fn-popover-button-margin;
+
+    button:not(:last-child) {
+      @include fn-set-margin-right($fn-popover-button-margin);
+    }
+  }
+
+  &__title,
+  &__content,
+  &__button-container {
+    &--no-padding {
+      padding: 0;
+    }
+  }
+
+  &__arrow {
+    @include fn-reset();
+    @include fn-set-square($fn-popover-arrow-size);
+
+    content: " ";
+    position: absolute;
+    background: $fn-color-white;
+    bottom: $fn-popover-arrow-offset;
+    transform: rotate(45deg);
+
+    &--bottom,
+    &--bottom-start,
+    &--bottom-end {
+      bottom: $fn-popover-arrow-offset;
+    }
+
+    &--bottom,
+    &--top {
+      right: $fn-popover-arrow-middle-calc;
+    }
+
+    &--bottom-start,
+    &--top-start {
+      right: auto;
+    }
+
+    &--bottom-end,
+    &--top-end {
+      @include fn-set-position-right($fn-popover-arrow-margin);
+    }
+
+    &--top,
+    &--top-start,
+    &--top-end {
+      top: $fn-popover-arrow-offset;
+    }
+
+    &--start,
+    &--start-top,
+    &--start-bottom {
+      @include fn-set-position-left($fn-popover-arrow-offset);
+    }
+
+    &--start,
+    &--end {
+      top: $fn-popover-arrow-middle-calc;
+    }
+
+    &--start-top,
+    &--end-top {
+      top: $fn-popover-arrow-margin;
+    }
+
+    &--start-bottom,
+    &--end-bottom {
+      bottom: $fn-popover-arrow-margin;
+    }
+
+    &--end,
+    &--end-top,
+    &--end-bottom {
+      @include fn-set-position-right($fn-popover-arrow-offset);
+    }
+  }
```

Comment:
All thee modifier classes need to be documented in the page header, including the options for no padding in the body and footer container, title truncation, etc.